src/pages/index.js

```diff
@@ -65,34 +109,137 @@ profileEditBtn.addEventListener("click", () => {
   editModal.open();
 });
 
-//--------------------ADD CARD POPUP-WITH-FORM INSTANTIATION-------------------->>
+//--------------------PROFILE PICTURE POPUP-WITH-FORM INSTANTIATION-------------------->>
+
+const profilePictureModal = new PopupWithForm(
+  "#profile-picture-edit-modal",
+  () => {
+    profilePictureModal.loadingButtonState();
+    api
+      .setProfilePicture(profilePictureFormInput.value)
+      .then((res) => {
+        profileAvatar.src = res.avatar;
+        profilePictureModal.resetButtonState();
+      })
+      .catch((err) => {
+        api.logError(err);
+      });
+    profilePictureModal.close();
+  },
+  "Save"
+);
+
+profilePictureModal.setEventListeners();
 
-const addModal = new PopupWithForm("#profile-add-modal", () => {
-  const { title, url } = addModal.getInputValues();
-  const newCard = renderCard({ name: title, link: url });
-  cardSection.addItem(newCard);
-  addModal.close();
+profilePictureEditBtn.addEventListener("click", () => {
+  profilePictureModal.open();
 });
 
+//--------------------ADD CARD POPUP-WITH-FORM INSTANTIATION-------------------->>
+
+const addModal = new PopupWithForm(
+  "#profile-add-modal",
+  () => {
+    addModal.loadingButtonState();
+    const { title, url } = addModal.getInputValues();
```

Comment:
`    const { title, url } = addModal.getInputValues();
`

You don't need to call `getInputValues` in `index.js` because you get all the values in line 142

```js
  ({ title, url }) => {
    addModal.loadingButtonState();
    api
      .addNewCard({ name: title, link: url })

```